packages/baobab.rb

```diff
@@ -36,8 +37,4 @@ def self.build
   def self.install
     system "DESTDIR=#{CREW_DEST_DIR} ninja -C builddir install"
   end
-
-  def self.postinstall
-    system "update-mime-database #{CREW_PREFIX}/share/mime"
-  end
```

Comment:
This is the only package we do this?

Reply:
I didn't want to edit the others until I got consensus on the approach.

Reply:
more can be added as we update...